rllib/evaluation/worker_set.py

```diff
@@ -281,18 +296,19 @@ def _get_spaces_from_remote_worker(self):
             for e in remote_spaces[0]
         }
 
-        # Try to add the actual env's obs/action spaces.
-        env_spaces = self.foreach_worker(
-            lambda worker: worker.foreach_env(
-                lambda env: (env.observation_space, env.action_space)
-            ),
-            remote_worker_ids=[worker_id],
-            local_worker=False,
-        )
-        if env_spaces:
-            # env_spaces group spaces by environment then worker.
-            # So need to unpack thing twice.
-            spaces["__env__"] = env_spaces[0][0]
+        if not issubclass(self.env_runner_cls, EnvRunner):
```

Comment:
Can we add a comment here that explains why we don't have to do this for EnvRunner anymore. Is it b/c we don't have Policies anymore?

Reply:
What I mean is: How does EnvRunner know, what spaces to build the RLModules with? Does it get it from the config already directly (through the MARLModuleSpec)?

Reply:
As I understand it `RLModule`s have a space for which they build their models. Is there something else needed? Could there be inactive models? Or could an `EnvRunner` sample only for one of several modules?